### ISSUE TITLE
Style attribute on element inside a group should overwrite group style attribute

### ISSUE THREAD

**Issue by dubert** [magicismight/react-native-svg#85]
For example if I write:
```
<G fill='none' fillRule='evenodd'>
    <Path id='Charles' fill='blue' d='M25 46C13.402 46 4 36.598 4 25c0-.124 0-.247.003-.37 10.914-1.558 14.883.57 22.014 2.304 7.13 1.733 12.48 1.17 19.325.355.19-.024.367-.03.537-.02C44.746 37.8 35.83 46 25 46z'/>
    <Circle cx={25} cy={25} r={23.5} stroke={C_BLACK} strokeWidth={3} />
    <Circle cx={25} cy={25} r={21.5} stroke={C_WHITE} strokeWidth={3} />
  </G>
```
then the Path `Charles` does not have a fill. Instead I have to write:
```
<G fillRule='evenodd'>
    <Path id='Charles' fill='blue' d='M25 46C13.402 46 4 36.598 4 25c0-.124 0-.247.003-.37 10.914-1.558 14.883.57 22.014 2.304 7.13 1.733 12.48 1.17 19.325.355.19-.024.367-.03.537-.02C44.746 37.8 35.83 46 25 46z'/>
    <Circle fill='none' cx={25} cy={25} r={23.5} stroke={C_BLACK} strokeWidth={3} />
    <Circle fill='none' cx={25} cy={25} r={21.5} stroke={C_WHITE} strokeWidth={3} />
  </G>
```
which is slightly more tedious and not how SVG works natively.

**Comment by magicismight** [comment]
Thanks for reporting this.
This is a known bug and had it fixed in `definations-refactor` branch on iOS.
Will be released after finished Android side.

**Comment by dubert** [comment]
You're awesome. Thanks for making and maintaining such a great library!

**Comment by magicismight** [comment]
Fixed in 4.0.0-rc
I`m closing this.
